geopandas/io/tests/test_arrow.py

```diff
@@ -493,3 +494,19 @@ def test_feather_arrow_version(tmpdir):
         ImportError, match="pyarrow >= 0.17 required for Feather support"
     ):
         df.to_feather(filename)
+
+
+def test_write_read_parquet():
+    gdf = geopandas.GeoDataFrame(geometry=[box(0, 0, 10, 10)], crs="epsg:4326")
+    test_file = "~/test_file.parquet"
+    gdf.to_parquet(test_file)
+    geopandas.read_parquet(test_file)
+    os.remove(os.path.expanduser(test_file))
+
+
+def test_write_read_feather():
+    gdf = geopandas.GeoDataFrame(geometry=[box(0, 0, 10, 10)], crs="epsg:4326")
+    test_file = "~/test_file.feather"
+    gdf.to_feather(test_file)
+    geopandas.read_feather(test_file)
```

Comment:
```suggestion
    f_df = geopandas.read_feather(test_file)
    assert_geodataframe_equal(df, f_df, check_crs=True)
```

Reply:
Similar to previous comment, but this time applied to the suggested change. Shouldn't the first arg be `gdf` rather than `df`?